tests/ps.go

```diff
@@ -0,0 +1,180 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package tests
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/onsi/ginkgo/v2"
+	"github.com/onsi/gomega"
+	"github.com/runfinch/common-tests/command"
+	"github.com/runfinch/common-tests/option"
+)
+
+// Ps tests functionality of `ps` command.
+func Ps(o *option.Option) {
+	sha256RegexTruncated := `^[a-f0-9]{12}$`
+	sha256RegexFull := `^[a-f0-9]{64}$`
+
+	containerNames := []string{"ctr_1", "ctr_2"}
+	ginkgo.Describe("Ps command", func() {
+		pwd, _ := os.Getwd()
+		ginkgo.BeforeEach(func() {
+			command.RemoveAll(o)
+			command.Run(o, "network", "create", testNetwork)
+			command.Run(o, "run", "-d",
+				"--name", containerNames[0],
+				"--label", "color=red",
+				"-v", fmt.Sprintf("%s:%s", pwd, pwd),
+				"-w", pwd,
+				defaultImage)
+			command.Run(o, "run", "-d",
+				"--label", "color=green",
+				"--network", testNetwork,
+				"-p", "8081:80",
+				"--name", containerNames[1],
+				defaultImage, "sleep", "infinity")
```

Comment:
Maybe we can move the initialization into each subject node? Different subject nodes need different things (e.g., `"-p", "8081:80"`, `"--label", "color=green"`, etc.), and I feel that currently we're coupling the test data together, which may make it harder to maintain in the future. Also, it can make it harder to read as right now one needs to scroll up to see the setup when reviewing things like `gomega.Expect(psOutput).Should(gomega.ContainSubstring("8081"))`.

Reply:
Good point. I have split the test setup into two groups. One is for the filters and the other one is for generic checks. Pls let me know if you want me to split it even more. I am in favor of keeping all the filter in one place as it reduces the code lines quite a bit.